app/assets/stylesheets/component-library-overrides.css

```diff
@@ -12,14 +12,17 @@
 }
 
 .navbar-logo {
-  --navbar-logo-width: 360px;
-  height: 34px;
-  width: var(--navbar-logo-width);
+  --sul-logo-width: 360px;
+  --sul-logo-height: 34px;
+
+  &.stacked-mobile {
+    --sul-logo-height: 58px;
```

Comment:
I'm not sure how to test this with a local version of the component library, but does this need a `@media` query too so it only applies at the xs(?) breakpoint?

Reply:
@cbeer The library website applies to the small breakpoint and down

Reply:
In order to test locally change "https://cdn.jsdelivr.net/gh/sul-dlss/component-library@v2025-06-10/styles/sul.css" to "https://sul-dlss.github.io/component-library/preview-146/styles/sul.css"